loaders/authentication.js

```diff
@@ -0,0 +1,38 @@
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
+
+//base URL of cigarra authentication
+const URL = 'https://sigarra.up.pt/feup/pt/mob_val_geral.autentica?'
+
+//build URL with credentials
+let urlRequest = URL + 'pv_login=' + process.env.UP_USER + '&pv_password=' + process.env.UP_PWD;
+
+let cookie = null;
+
+//Called sigarra api to make authentication
+async function updateCookie(){
+   try {
+        const response = await fetch(urlRequest, {
+            method: 'POST'
+        });
+        if(response.ok){
+            return await response.json();
+        }
+        throw new Error('Request Failed: ' + response.status + "-" + response.statusText);
+
+   }catch (error){
+       console.log(error);
+   }
+}
+
+//give value to cookie
+async function getCookie(){
+    if(cookie == null){
+        cookie = await updateCookie();
+    }
+    return cookie
+}
+
+module.exports = getCookie;
+
```

Comment:
The placement of this file is something I disagree with a lot :slightly_smiling_face: 

The `loaders` directory, as per [`nijobs-be`'s project structure](https://github.com/NIAEFEUP/nijobs-be#project-structure) is for code that will run when the server starts up. The way this code is currently structured, it might fit as a middleware or, more likely, as a "helper" file. Moving it to `helpers/auth(entication).js` would make much more sense to me.